src/Draco.Compiler/Internal/Binding/LocalBinder.cs

```diff
@@ -169,7 +169,9 @@ private void Build()
     {
         FunctionDeclarationSyntax function => new SourceFunctionSymbol(this.ContainingSymbol, function),
         ParameterSyntax parameter => new SourceParameterSymbol((FunctionSymbol)this.ContainingSymbol, parameter),
-        VariableDeclarationSyntax variable => new SourceLocalSymbol(this.ContainingSymbol, new TypeVariable(localCount), variable),
+        VariableDeclarationSyntax variable => variable.FieldKeyword != null ?
+            new SourceFieldSymbol((TypeSymbol)this.ContainingSymbol, variable) :
```

Comment:
We should not be able to define fields in a local context at all. Again, the props PR might be needed, it deals with these cases.

Reply:
I don't have to do anything then ?

Reply:
Yep, leave og. code